build/yamls/antrea-eks-node-init.yml

```diff
@@ -0,0 +1,76 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-node-init
+  name: antrea-node-init
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-node-init
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-node-init
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - name: node-init
+          image: gcr.io/google-containers/startup-script:v1
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          env:
+          - name: STARTUP_SCRIPT
+            value: |
+              #! /bin/bash
+
+              set -o errexit
+              set -o pipefail
+              set -o nounset
+
+              if [ -f /opt/cni/antrea-node-init-status ]; then
+                  echo "Antrea node init already done. Exiting"
+                  exit
+              fi
+
+              while true; do
+                  cni_conf=$(ls /etc/cni/net.d | head -n1)
+                  if [[ ! -z $cni_conf ]]; then break; fi
+                  echo "Waiting for cni conf file"
+                  sleep 2s
+              done
+              cni_conf="/etc/cni/net.d/$cni_conf"
+
+              while true; do
+                  if grep -sq "antrea" $cni_conf; then break; fi
+                  echo "Waiting for antrea config to be updated"
+                  sleep 2s
+              done
+
+              # Wait for additional 5 sec for cni config file update
```

Comment:
I think this comment would benefit from including more context regarding the reason, like the link that Quan shared. We do not wait for a file update here, we wait for kubelet to register the file update.

Reply:
done